dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -198,22 +193,23 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                     {
                         wil::unique_cotaskmem_string unpackagedAppUserModelId;
                         RegisterUnpackagedApplicationHelper(remoteId, unpackagedAppUserModelId);
```

Comment:
Add a comment here that unpackagedAppUserModelId is generated by the Long Running Process singleton.